### DISCUSSION TITLE
"Trying to access array offset on value of type bool" when using with Laravel Vapor

### DISCUSSION THREAD

**Discussion by phuclh** [spatie/laravel-rate-limited-job-middleware#41] [Q&A]
I got this error `Trying to access array offset on value of type bool` when using this package on Vapor (I had Redis installed). It works perfectly in my local. Does anyone have this issue before? Thank you!
<img width="1975" alt="CleanShot 2022-08-11 at 08 08 10@2x" src="https://user-images.githubusercontent.com/6707194/184166450-4f85c208-8921-4e50-88b4-86b6c6da9825.png">

**Comment by freekmurze** [comment]
If you find the bug, I'd appreciate a PR that fixes it. If possible also include a test.
